Fix build failure
>   CMake Error at third_party/proton/test/unittest/TraceDataIO/CMakeLists.txt:4 (add_triton_ut):
>    Unknown CMake command "add_triton_ut"

From https://github.com/triton-lang/triton/pull/8237

diff --git a/third_party/proton/test/CMakeLists.txt b/third_party/proton/test/CMakeLists.txt
index e250d49f282e..5b05b089445d 100644
--- a/third_party/proton/test/CMakeLists.txt
+++ b/third_party/proton/test/CMakeLists.txt
@@ -1 +1,3 @@
-add_subdirectory(unittest)
+if(TRITON_BUILD_UT)
+  add_subdirectory(unittest)
+endif()
